apps/evm/src/app/[lang]/(bridge)/stake/components/EvmStrategy/EvmStrategy.tsx

```
@@ -0,0 +1,43 @@
+  const [tab, setTab] = useState<Tab>(Tab.Deposit);
+
+  const isLending = strategy?.meta.type === 'lending';
+
+  const depositTitle = isLending ? <Trans>Supply</Trans> : <Trans>Stake</Trans>;
+  const withdrawTitle = isLending ? <Trans>Withdraw</Trans> : <Trans>Unstake</Trans>;
+
+  return (
+    <Flex direction='column'>
+      <Tabs fullWidth selectedKey={tab} size='lg' onSelectionChange={(key) => setTab(key as Tab)}>
+        <TabsItem key={Tab.Deposit} title={strategy ? depositTitle : <Skeleton height='xl' width='6rem' />}>
+          <></>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)